Create 2024-06-12.md (#155)
department-of-veterans-affairs/developer-portal-content: content/verification/veteran-verification/release-notes/2024-06-12.md

> We updated the results returned for disability ratings.

Reviewer: ```suggestion
We've updated the results returned for disability ratings.
```

> * They continue to exclude ratings that do not have a “begin” date, with the exception of ratings with "Not Service Connected" and "1151 Service Denied" decisions.

Reviewer: ```suggestion
- They continue to exclude ratings that do not have a “begin” date.
- This does not include ratings with "Not Service Connected" and "1151 Service Denied" decisions.
```
